src/view/components/extensionView.jsx

```diff
@@ -10,76 +10,170 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import React, { useState, useRef } from "react";
+import React, { useState, useRef, useEffect } from "react";
 import PropTypes from "prop-types";
+
+import { useFormik, FormikProvider } from "formik";
+import { object } from "yup";
+import { ProgressCircle } from "@adobe/react-spectrum";
 import useExtensionBridge from "../utils/useExtensionBridge";
-import ExtensionViewContext from "./extensionViewContext";
+import useReportAsyncError from "../utils/useReportAsyncError";
+import FillParentAndCenterChildren from "./fillParentAndCenterChildren";
 
-// This component wires up Launch's extension bridge, and creates the
-// ExtensionViewContext. It should be used for each view inside an extension.
-const ExtensionView = ({ render }) => {
+const ExtensionView = ({
+  render,
+  getInitialValues,
+  getSettings,
+  validateFormikState,
+  formikStateValidationSchema,
+  validateNonFormikState
+}) => {
```

Comment:
I like that you support both props and `registerImperativeFormApi`. That's kind of nice.